src/app/home/home.component.html

```diff
@@ -0,0 +1,4 @@
+<div class="container">
+    <img src="../../assets/img/fundo.jpg" alt="Snow" style="width:100%;">
```

Comment:
```suggestion
    <img src="../../assets/img/fundo.jpg" alt="Carambola" style="width:100%;">
```
Sei que é apenas um placeholder, mas já podemos colocar algo mais neutro no alt text